### PR TITLE
Deprecate qualys v1

### PR DESCRIPTION
**Pull request by PA-Rweins**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/38985

## Description
Deprecated old Qualys integration

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/qualys/Integrations/integration-qualys.yml [hunk 1]
```diff
@@ -2,30 +2,30 @@ commonfields:
...
-description: Network Security and Vulnerability Management
+description: Deprecated. Use Qualysv2 instead.
```
Comment (teizenman) [inline Packs/qualys/Integrations/integration-qualys.yml:8]:
```suggestion
description: Deprecated. Use Qualys v2 instead.
```

File: Packs/qualys/Integrations/integration-qualys.yml [hunk 2]
```diff
@@ -975,1476 +1003,2012 @@ script:
...
+    - name: scan_ref
+      required: true
+      description: 'The scan reference for a compliance scan. This will have the format:
+        compliance/nnnnnnnnnn.nnnnn'
+    description: fetch scan results for a scan
+    deprecated: true
 hidden: false
 fromversion: 5.0.0
+deprecated: true
+tests: No test
```
Comment (teizenman) [inline Packs/qualys/Integrations/integration-qualys.yml:3014]:
```suggestion
tests: 
- No tests
```

**Review by teizenman — CHANGES_REQUESTED** [review]
What about version bump? RN?

File: Packs/qualys/Integrations/integration-qualys.yml (hunk 1 above)
Comment (PA-Rweins) [reply Packs/qualys/Integrations/integration-qualys.yml:8]: fixed

Comment (PA-Rweins) [reply Packs/qualys/Integrations/integration-qualys.yml:8]: fixed

File: Packs/qualys/Integrations/integration-qualys.yml (hunk 2 above)
Comment (PA-Rweins) [reply Packs/qualys/Integrations/integration-qualys.yml:3014]: fixed

File: Packs/qualys/Integrations/integration-qualys.yml [hunk 3]
```diff
@@ -3011,4 +3011,5 @@ script:
 hidden: false
 fromversion: 5.0.0
 deprecated: true
-tests: No test
+tests:
+- No test
```
Comment (teizenman) [inline Packs/qualys/Integrations/integration-qualys.yml:3015]:
```suggestion
- No test
deprecated: true
```

**Review by teizenman — CHANGES_REQUESTED** [review]

File: Packs/qualys/ReleaseNotes/1_0_5.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Qualys (Deprecated)
+- Deprecated. Use **Qualys v2** integration instead.
```
Comment (amshamah419) [inline Packs/qualys/ReleaseNotes/1_0_5.md:4]:
```suggestion
- Deprecated. Use the **Qualys v2** integration instead.
```

Comment (PA-Rweins) [reply Packs/qualys/ReleaseNotes/1_0_5.md]: Fixed

**Review by teizenman — APPROVED** [review]

**Pull request merged**